tests/console/salt.pm

```diff
@@ -42,7 +42,7 @@ sub run {
 
     quit_packagekit;
     my @packages = qw(salt-master);
-    push @packages, 'salt-minion' unless is_jeos && (is_sle || is_leap);
+    push @packages, 'salt-minion' unless is_jeos && (is_sle || is_leap) && !is_community_jeos;
```

Comment:
At this point this needs a comment, something like `# On SLE based Minimal-VM/Minimal-Image, salt-minion has to be preinstalled`

Reply:
added, thanks